packages/gatsby-plugin-mdx/gatsby/source-nodes.test.js

```diff
@@ -0,0 +1,178 @@
+const { graphql } = require(`gatsby/graphql`)
```

Comment:
it seems like this test requires mocking a lot of code, making it prone to breakage and extra maintenance burden in the future. Have any ideas to simplify this?

Reply:
This test was mostly imported from `gatsby-transfomer-remark`. I'll try to find the way to test GraphQL query easier...